Cargo.toml

```diff
@@ -44,6 +44,7 @@ ion-c-sys = { path = "./ion-c-sys", version = "0.4" }
 [dev-dependencies]
 # Used by ion-tests integration
 walkdir = "^2.3"
+test-generator = "0.3"
```

Comment:
This is a `proc_macro` that allows us to get a test case per file.  This is super important because often there will be more than one failure and chasing that down within a super-large test case is problematic (especially with the debugger).